reviews/templates/reviews/review_detail_sidebar.html

```diff
@@ -103,6 +103,11 @@ <h3>{% trans "Details" %}</h3>
                 </strong>
             </li>
         {% endif %}
+        {% if review.email_checkbox %}
+            <li>
+                <b style='color:red;'>CHECK EMAIL</b>
```

Comment:
```suggestion
                <strong class="text-danger">{% trans "Extra info via e-mail!" %}</strong>
```
I prefer this. Try to use bootstrap classes when possible so we maintain a consistent look.

I would translate this with "Extra info via e-mail" as well haha.